{
  return new RuntimeException(msg,cause);
}
